packages/apps/crm/src/i18n/en.json

```diff
@@ -86,6 +86,16 @@
   "Crm_NoOpportunityAssociated": "No opportunity is associated",
   "Crm_LastOpportunity": "Last opportunity",
   "Crm_SocialNetworks": "Social networks",
+  "Crm_CreateEvent": "Create Event",
+  "Crm_Subject": "Subject",
+  "Crm_Type": "Type",
+  "Crm_Status": "Status",
+  "Crm_Partner": "Partner",
+  "Crm_StartDate": "Start date",
+  "Crm_EndDate": "End date",
+  "Crm_AllDay": "All day",
```

Comment:
Both french and english translations are not consistent, french is a question where english is not. Both translation should have the same meaning